src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -237,7 +237,11 @@ public static class TextCatalog
         /// <param name="inputColumn">The column containing text to remove stop words on.</param>
         /// <param name="outputColumn">The column containing output text. Null means <paramref name="inputColumn"/> is replaced.</param>
         /// <param name="language">Langauge of the input text column <paramref name="inputColumn"/>.</param>
-        public static StopWordsRemovingEstimator RemoveStopWords(this TransformsCatalog.TextTransforms catalog,
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        ///  [!code-csharp[FastTree](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/StopWordRemoverTransform.cs)]
+        /// ]]></format>
```

Comment:
wrap it with <example>:


/// <example>
        /// <format type="text/markdown">
        /// <![CDATA[
        /// [!code-csharp[StopWords](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/StopWordRemoverTransform.cs)]
        /// ]]>
        /// </format>
        /// </example>

Reply:
Thank you!